drivers/sensor/adxl372/adxl372.h

```diff
@@ -282,11 +282,8 @@ struct adxl372_data {
 	struct device *bus;
 #ifdef CONFIG_ADXL372_SPI
 	struct spi_config spi_cfg;
-#endif
-#if defined(CONFIG_ADXL372_SPI_GPIO_CS)
```

Comment:
Why did you remove this instead of replace it with `DT_ADI_ADXL372_0_CS_GPIO_CONTROLLER`? Same question below on line 322

Reply:
I removed it thinking that one day we might support multiple sensors and than you'd need it always defined.  I'm happy to keep the ifdef for now.

Reply:
Not necessarily. You wouldn't need it if you had multiple instances and both used hardware-supported CS. Let's keep the ifdef for now.

Reply:
Updated to put ifdef back.